src/utils/devtools.ts

```diff
@@ -30,7 +36,8 @@ export function devtools<T extends object>(proxyObject: T, name?: string) {
   }
 
   let isTimeTraveling = false
-  const devtools = extension.connect({ name })
+  const devtools: ReturnType<NonNullable<typeof extension>['connect']> =
+    extension.connect({ name })
```

Comment:
I guess we can remove the annotation.
```suggestion
  const devtools = extension.connect({ name })
```